src/lib/options/options-item.js

```diff
@@ -96,16 +95,15 @@ export default class OptionItem {
    * Saves an option value to the local storage.
    */
   save () {
-    let option = {}
+    const option = {} // eslint-disable-line no-prototype-builtins
```

Comment:
As I could see its properties would be added/changed - so it shouldn't be const.

Reply:
That's a good catch! Will change that